### PR TITLE
Add questionnaire id and remove case id

### PR DESCRIPTION
**Pull request by iwootten**
This PR adds the questionnaire id to the claims to launch eq with and removes the case id.

It is to be used to test the following PR: https://github.com/ONSdigital/eq-survey-runner/pull/2084

https://trello.com/c/0fkw3X5F/2984-jwt-claims-questionnaireid-and-caseid

### CODE REVIEW

File: launch.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 
 	urlValues.Add("ru_ref", authentication.GetDefaultValues()["ru_ref"])
 	urlValues.Add("collection_exercise_sid", uuid.NewV4().String())
-	urlValues.Add("case_id", uuid.NewV4().String())
```
Comment (MebinAbraham) [inline launch.go:150]: Do we want to keep this anyway since it is optional now in runner.

Comment (iwootten) [reply launch.go:150]: The card seems to specify that it should replace case_id

**Comment by jonnyshaw89** [comment]
We should migrate this PR to https://github.com/ONSdigital/eq-questionnaire-launcher

File: launch.go (hunk 1 above)
Comment (jonnyshaw89) [reply launch.go:150]: The card says it's no longer mandatory. therefore we should keep it and only send it if a value is provided.

**Comment by iwootten** [comment]
> We should migrate this PR to https://github.com/ONSdigital/eq-questionnaire-launcher

It has been already: https://github.com/ONSdigital/eq-questionnaire-launcher/pull/2

**Pull request closed without merging**